queries.sql

```diff
@@ -1,3 +1,61 @@
-/*Queries that provide answers to the questions from all projects.*/
+-- * Queries that provide answers to the questions from all projects.
+-- ** Project 1 START.
+SELECT
+    *
+FROM
+    animals
+WHERE
+    name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT
+    name
+from
+    animals
+WHERE
+    date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT
+    name
+FROM
+    animals
+WHERE
+    neutered
+    AND escape_attempts < 3;
+
+SELECT
+    date_of_birth
+FROM
+    animals
+WHERE
+    name IN ('Agumon', 'Pikachu');
+
+SELECT
+    name,
+    escape_attempts AS escapes
+FROM
+    animals
+WHERE
+    weight_kg > 10.5;
+
+SELECT
+    *
+FROM
+    animals
+WHERE
+    neutered;
```

Comment:
In both cases of the `neutered` column, I am implicitly checking its condition, which is a shorthand way to check for truthiness in PostgreSQL. It is equivalent to `neutered` **= TRUE** or `neutered` **IS TRUE**. This is because PostgreSQL, and many other SQL databases, consider a Boolean column without a specific equality or truth check, to be equivalent to checking if it's **TRUE**. Like when coding a boolean variable in an `if` statement often follows a similar pattern 😃.

Reply:
okay, weldone